### PR TITLE
Fix CI failures in master

### PR DESCRIPTION
**Pull request by bvaughn**
Fix some breakage that occurred in master after #9449 was merged.

This is just a bandaid fix so CI stops failing. I left a TODO comment for us to follow up on.

### CODE REVIEW

File: src/__mocks__/UIManager.js [hunk 1]
```diff
@@ -90,25 +93,22 @@ var RCTUIManager = {
...
-    parentTag,
-    moveFromIndices = [],
-    moveToIndices = [],
-    addChildReactTags = [],
-    addAtIndices = [],
-    removeAtIndices = [],
-  ) {
+  manageChildren: jest.fn(function manageChildren(parentTag, moveFromIndices = [
+  ], moveToIndices = [], addChildReactTags = [], addAtIndices = [
+  ], removeAtIndices = []) {
```
Comment (gaearon) [inline src/__mocks__/UIManager.js]: cc @vjeux, isn’t very pretty

Comment (bvaughn) [inline src/__mocks__/UIManager.js]: 😆  Agreed

**Comment by flarnie** [comment]
My mistake - you added the TODO, and that is fine.

File: src/__mocks__/UIManager.js (hunk 1 above)
Comment (vjeux) [inline src/__mocks__/UIManager.js]: I fixed this yesterday. I'll do a release this morning with the fix. Sorry about this!

It's going to look like this:
```js
  manageChildren: jest.fn(function manageChildren(
    parentTag,
    moveFromIndices = [],
    moveToIndices = [],
    addChildReactTags = [],
    addAtIndices = [],
    removeAtIndices = []
  ) {
    autoCreateRoot(parentTag);
```

Comment (bvaughn) [inline src/__mocks__/UIManager.js]: So much nicer!

Comment (gaearon) [inline src/__mocks__/UIManager.js]: Always one step ahead of us.

Comment (vjeux) [inline src/__mocks__/UIManager.js]: https://github.com/prettier/prettier/releases/tag/1.2.0

**Pull request merged**